platform/darwin/src/MGLRasterSource.h

```diff
@@ -1,18 +1,57 @@
 #import "MGLSource.h"
 #import "MGLTypes.h"
 
+#import <CoreGraphics/CoreGraphics.h>
+
 @class MGLTileSet;
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+ A raster tile source.
+ 
+ @see <a href="https://www.mapbox.com/mapbox-gl-style-spec/#sources-raster">The
```

Comment:
This is fine for now, but ideally we would copy the relevant documentation here, adapt it to Cocoa terminology, and not even need to link to the style specification.